src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/mvn.cpp

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/mvn.hpp"
+#include "shared_test_classes/single_layer/mvn.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace MVN {
+
+const std::vector<bool> normalizeVariance = {
+       true
+};
+
+std::vector<ElementType> inpPrc = {
+        ElementType::i8,
+        ElementType::f32,
+};
+std::vector<ElementType> outPrc = {
+        ElementType::f32,
+};
+
+std::vector<CPUSpecificParams> cpuParams_4D = {
+        CPUSpecificParams({nchw}, {nchw}, {}, {}),
+        CPUSpecificParams({nhwc}, {nhwc}, {}, {}),
+};
+
+std::vector<CPUSpecificParams> cpuParams_5D = {
+        CPUSpecificParams({ncdhw}, {ncdhw}, {}, {}),
+        CPUSpecificParams({ndhwc}, {ndhwc}, {}, {}),
+};
+
+std::vector<fusingSpecificParams> fusingParamsSet {
+        emptyFusingSpec,
+};
+
+std::vector<fusingSpecificParams> fusingParamsSetStaticShape {
+        emptyFusingSpec,
+};
+
+const auto Mvn3D = ::testing::Combine(
+       ::testing::Combine(
+           ::testing::ValuesIn(inputShapes_3D()),
+           ::testing::Values(ElementType::f32),
+           ::testing::ValuesIn(emptyReductionAxes()),
+           ::testing::ValuesIn(acrossChannels()),
+           ::testing::ValuesIn(normalizeVariance),
+           ::testing::ValuesIn(epsilon())),
+       ::testing::Values(emptyCPUSpec),
+       ::testing::ValuesIn(fusingParamsSet),
+       ::testing::ValuesIn(inpPrc),
+       ::testing::ValuesIn(outPrc));
+
+INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_Mvn3D, MvnLayerCPUTest, Mvn3D, MvnLayerCPUTest::getTestCaseName);
+
+const auto Mvn4D = ::testing::Combine(
+       ::testing::Combine(
+               ::testing::ValuesIn(inputShapes_4D()),
+               ::testing::Values(ElementType::f32),
+               ::testing::ValuesIn(emptyReductionAxes()),
+               ::testing::Values(true),
```

Comment:
could you please add a comment here, like:
::testing::Values(true), // "initAcrossChannels = false" is not supported (or fails by accuracy?) for NHWC tensor.
Please also add the similar comment to the code where the check happens.
Also, it seems the check in isSupported method only cares about NHWC layout.
Here we are not testing the combination HCHW + initAcrossChannels = false for some reason.

Reply:
Added missed case "HCHW + initAcrossChannels = false" for 4D and 5D

Reply:
As discussed, proposed to leave this case in common and write test-local check to skip the test for arm in case this combination appeared.